### PR TITLE
#318 - ArcGIS 10.1 Parameters

### PR DESCRIPTION
**Pull request by xlhomme**
Implements #318.  Originally submitted as #223 by @ximex

### CODE REVIEW

**Comment by smithkm** [comment]
A whole lot of pointless whitespace changes crept in somewhere along with some changes from line to block comments.  I think you may have run some sort of auto-formatting tool.  That's strongly discouraged as it makes the commit history hard to follow.

You should also squash/fixup your commits together to make them clearer.

**Comment by xlhomme** [comment]
Tested with success an exploded arcgis cache produced with 10.0.

**Comment by smithkm** [comment]
@xlhomme Could you fix the whitespace issues and squash your commits?

**Comment by xlhomme** [comment]
Ok I could try tomorrow morning.

**Comment by smithkm** [comment]
Squashed the commits and merged manually.
